_includes/install_instructions/python.html

```diff
@@ -100,9 +135,11 @@ <h4>Video Tutorial</h4>
             for the files.
             Type <code>yes</code> and press
             <kbd>Enter</kbd> (or <kbd>Return</kbd>)
-            to prepend Anaconda to your <code>PATH</code>
-            (this makes the Anaconda distribution the default Python).
+            to prepend Miniforge to your <code>PATH</code>
+            (this makes the Miniforge distribution the default Python).
           </li>
+	  <li>Download the <a href="./data/swc_environment.yml" download>environment file</a>.</li>
+	  <li>Search for the application "Miniforge Prompt", open it and run: <code>conda env create -f ~/Downloads/swc_environment.yml</code></li>
```

Comment:
How confident can we be that `~/Downloads/` is the location where everyone will find the environment file after downloading it? I feel that we need to provide additional details here to ensure that learners save it to the right place.

Reply:
```suggestion
	  <li>Download the <a href="./data/swc_environment.yml" download>environment file</a>. Save the file to your Downloads folder.</li>
	  <li>Search for the application "Miniforge Prompt", open it and run: <code>conda env create -f ~/Downloads/swc_environment.yml</code></li>
```